tests_e2e/tests/scripts/ext_sequencing-get_ext_enable_time.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env pypy3
```

Comment:
since this script is just checking the test log, a small file. something you may want to consider is to copy the file to the orchestrator machine (use SshClient.copy_from_node) and process the file locally.

Not required, just pointing it out, since that may facilitate debugging, error handling, etc